src/spaceObjects/missiles/missileWeapon.cpp

```diff
@@ -1,12 +1,13 @@
 #include "missileWeapon.h"
 #include "particleEffect.h"
-#include "explosionEffect.h"
+#include "spaceObjects/explosionEffect.h"
 
-MissileWeapon::MissileWeapon(string multiplayerName, const MissileWeaponData& data)
-: SpaceObject(10, multiplayerName), data(data)
+MissileWeapon::MissileWeapon(string multiplayer_name, const MissileWeaponData& data)
+: SpaceObject(10, multiplayer_name), data(data)
 {
     target_id = -1;
     target_angle = 0;
+    category_modifier = 1;
```

Comment:
I think the category_modifier needs to be replicated to the clients as well, for proper rendering simulation on the client side.